src/hooks/useNotificationTypes.tsx

```diff
@@ -205,9 +206,72 @@ export const notificationTypes: NotificationTypeConfig[] = [
       return () => {};
     },
   },
+  {
+    type: NotificationType.ReleaseUpdates,
+    useTrigger: ({ trigger }) => {
+      const stringGetter = useStringGetter();
+      const body = (
+        <div>
+          {stringGetter({
+            key: 'NOTIFICATIONS.RELEASE_REWARDS_AND_FULL_TRADING.BODY',
+            params: {
+              BLOGPOST: (
+                <$Link
+                  href="https://www.dydxopsdao.com/blog/deep-dive-full-trading"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {stringGetter({ key: STRING_KEYS.HERE })}
+                </$Link>
+              ),
+              // todo: update localization to flip the two
+              DOS_BLOGPOST: (
+                <$Link
+                  href="https://dydx.exchange/blog/v4-full-trading"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {stringGetter({ key: STRING_KEYS.HERE })}
+                </$Link>
+              ),
+            },
+          })}
+        </div>
+      );
+
+      useEffect(() => {
+        trigger(
+          'rewards-and-full-trading-live',
+          {
+            icon: <Icon iconName={IconName.LogoShort} />,
+            title: stringGetter({ key: 'NOTIFICATIONS.RELEASE_REWARDS_AND_FULL_TRADING.TITLE' }),
+            renderCustomBody: ({ isToast, notification }) => (
```

Comment:
I don't think `ReleaseUpdatesNotification` is necessary. You should be able to just use `body` if you remove the `<div>` tags.